### PR TITLE
fix: Low accessibility metrics

### PR DESCRIPTION
**Pull request by flacial**
Closes https://github.com/garageScript/c0d3-app/issues/1891

This PR replaces `h3` headings with a `p` element and set the lang attribute to the `html` element as `en`.

The reason behind these changes can be found here:

- https://www.w3.org/WAI/tutorials/page-structure/headings/
- https://dequeuniversity.com/rules/axe/3.5/html-has-lang

Before:

image

After:

image

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ const moduleExports = {
   pageExtensions: ['tsx', 'js', 'jsx', 'mdx', 'ts'],
   images: {
     domains: ['cdn.discordapp.com']
+  },
```
Comment (anthonykhoa) [inline next.config.js:33]: what does this do?

Comment (flacial) [reply next.config.js:33]: It's used to config the languages the app will support and set a default one that will be used for all the routing.

File: components/LandingPage.tsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ const LandingPage: React.FC = () => {
               alt="Work Experience"
               layout="fixed"
             />
-            <h4 className="mt-3 fw-bold">Work Experience</h4>
+            <p className="mt-3 fw-bold h3">Work Experience</p>
```
Comment (anthonykhoa) [inline components/LandingPage.tsx:101]: Does using `p` instead of a header for this file cause any change in UI?

Comment (flacial) [reply components/LandingPage.tsx:101]: No, since it gets the `h3` header css from the `h3` classname.

**Review by SlyBouhafs — APPROVED** [review]

**Review by songz — APPROVED** [review]

**Review by anthonykhoa — APPROVED** [review]

**Pull request merged**